### ISSUE TITLE
Animated.createAnimatedComponent(TextInput) can not receive refs

### ISSUE THREAD

**Issue by emilioheinz** [facebook/react-native#27808]
I'm working on a new project where I need a custom and animated input, but to animate the bottom line of the input I need to create a animated component of a React Native `TextInput`, it works fine, but, when I try to pass a ref to that component using `React.forwardRef` the ref doesn't work.

I've tried without the `Animated.createAnimatedComponent(TextInput)` stuff and it works perfectly (but without animation).

The problem is that I don't have access to `ref.current` or any other function accessed from `current`.

The exact error when I try to focus the input via ref is:
```
ExceptionsManager.js:44 TypeError: animatedInputRef.current.focus is not a function
```

**React Native version:** 0.61.5 
```
❯ react-native info
info Fetching system and libraries information...
System:
    OS: macOS Mojave 10.14.5
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 3.81 GB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.14.1 - ~/.nvm/versions/node/v12.14.1/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.14.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-create-library: 3.1.2
```

## Steps To Reproduce

1. Start a new React Native project
2. Create a Input with `Animated.createAnimatedComponent(TextInput)`
3. Create a Screen
5. Create a ref using `useRef` and pass it to the Animated TextInput
6. Try to focus the Animated TextInput from the screen using the ref

**Describe what you expected to happen:**
I spect that the Animated component could receive a ref properly and could be focused without crashing the application.

**Snack, code example, screenshot, or link to a repository:**
Github repository with code example: https://github.com/emilioheinz/animated-text-input-ref-bug

----

**Some images to show the exact crash:**

<details>
<summary>Main example screen</summary>
<br>
Thats the main screen of the example provided on the github repository:

![image](https://user-images.githubusercontent.com/47385105/72680167-a621ea00-3a95-11ea-8bab-db67dfe90943.png)
</details>

<details>
<summary>When I press the button that tries to focus the animated input</summary>
<br>
Thats the error that happen when I try to focus the AnimatedTextInput via ref:

![image](https://user-images.githubusercontent.com/47385105/72680179-cb165d00-3a95-11ea-98f2-adb60167cb19.png)
</details>

<details>
<summary>When I press the button that focus the normal TextInput</summary>
<br>
Thats the example of the normal input working with the same logic of refs:

![image](https://user-images.githubusercontent.com/47385105/72680189-dbc6d300-3a95-11ea-8dd8-df11e456b159.png)
</details>

**Comment by TheSavior** [comment]
I believe you need to call

```
animatedInputRef.current.getNode().focus
```

This API was changed on master to be able to do what you want and skip the `getNode` call and will land in 0.63. You can see the new behavior here: https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/createAnimatedComponent.js#L126-L137

**Comment by chrise86** [comment]
Looks like this landed in 0.62 for anyone who stumbles across this.

https://reactnative.dev/blog/2020/03/26/version-0.62

**Comment by Knjaz89** [comment]
> Looks like this landed in 0.62 for anyone who stumbles across this.
> 
> https://reactnative.dev/blog/2020/03/26/version-0.62

How to install for animated.scrollView? and call scrollTo
